velox/functions/prestosql/DateTimeFunctions.h

```diff
@@ -118,23 +118,25 @@ FOLLY_ALWAYS_INLINE bool call(
 }
 VELOX_UDF_END();
 
-VELOX_UDF_BEGIN(hour)
-const date::time_zone* timeZone_ = nullptr;
+struct udf_hour {
```

Comment:
I like that you can see the outer struct wrapper, and you can see where the "udf_hour" symbol comes from.